tcp_offset_mm: [0.0, 0.0, 0.0, 0.0]   # [dx, dy, dz, yaw] of tool TCP relative to flange
speed_presets:
  pick_speed_mm_s: 150.0
  place_speed_mm_s: 150.0
  transit_speed_mm_s: 150.0
heights_mm:
  approach_above: 50.0
  lift_after_pick: 60.0
  place_above: 40.0
gripper:
  stroke_min_mm: 5.0
  stroke_max_mm: 80.0
  default_clearance_mm: 3.0
